<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyAudio Spectrum</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="spectrumGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>125</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QRadioButton" name="spectrum1RadioButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>8</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Spectrum 1</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="spectrum2RadioButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Spectrum 2</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="spectrum3RadioButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>52</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Spectrum 3</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="spectrum4RadioButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>74</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Spectrum 4</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="spectrum5RadioButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>96</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Spectrum 5</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="startFrequencyLabel">
         <property name="text">
          <string>Start Frequency [Hz]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="startFrequencySpinBox">
         <property name="maximum">
          <number>20000</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="stopFrequencyLabel">
         <property name="text">
          <string>Stop Frequency [Hz]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="stopFrequencySpinBox">
         <property name="maximum">
          <number>20000</number>
         </property>
         <property name="value">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="stepSizeLabel">
         <property name="text">
          <string>Step Size [Hz]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="stepSizeSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="timePerStepLabel">
         <property name="text">
          <string>Time per Step [ms]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="timePerStepSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignVCenter">
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="text">
          <string>Clear Spectrum</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layout">
      <property name="spacing">
       <number>6</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSaveSpectrumAs"/>
    <addaction name="actionOpenSpectrum"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionDevices"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionSignalWindow"/>
    <addaction name="actionFFTWindow"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindows"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionFFTWindow">
   <property name="text">
    <string>FFT of Microphone Signal</string>
   </property>
  </action>
  <action name="actionOpenSpectrum">
   <property name="text">
    <string>Open Spectrum...</string>
   </property>
  </action>
  <action name="actionSaveSpectrumAs">
   <property name="text">
    <string>Save Spectrum As...</string>
   </property>
  </action>
  <action name="actionSignalWindow">
   <property name="text">
    <string>Microphone Signal</string>
   </property>
  </action>
  <action name="actionMeasure_Wavefunction">
   <property name="text">
    <string>Measure Wavefunction</string>
   </property>
  </action>
  <action name="actionInput_Output_Device">
   <property name="text">
    <string>Input/Output Device</string>
   </property>
  </action>
  <action name="actionDevices">
   <property name="text">
    <string>In-/Output Devices</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
